draft-ietf-ppm-dap.md

```diff
@@ -1871,6 +1884,14 @@ The Leader then begins working with the Helper to aggregate the reports
 satisfying the query (or continues this process, depending on the VDAF) as
 described in {{aggregate-flow}}.
 
+Collection job creation MUST be idempotent. That is, multiple requests to
+`PUT /tasks/{tasks}/collection_jobs/{collection-job-id}` for the same
+`collection-job-id` and the same request body MUST succeed. However, changing
+a collection job's parameters is illegal, so further requests to
```

Comment:
Does this conflict with max_batch_query_count? if a collection-job-id has been queried more than max_batch_query_count, the Leader is suppose to fail the CollectionReq with "batchQueriedTooManyTimes".

Reply:
Bump

Reply:
This does not conflict with max batch query count. Each of the queries against a given batch is a distinct collection with a distinct collection ID.